ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -308,6 +308,10 @@ data instance BlockQuery (ShelleyBlock proto era) :: Type -> Type where
     :: CG.ConwayEraGov era
     => BlockQuery (ShelleyBlock proto era) (CG.RatifyState era)
 
+  GetFuturePParams
+    :: CG.ConwayEraGov era
+    => BlockQuery (ShelleyBlock proto era) (Maybe (LC.PParams era))
```

Comment:
`GetFuturePParams` isn't Conway-specific, the `ConwayEraGov` constraint is not necessary.

Reply:
Good point. I removed it.